docs/cow-protocol/tutorials/swap-in-safe-app/README.mdx

```diff
@@ -13,6 +13,22 @@ This is interesting for Safe Dapps because:
 
 The easiest way to get started is to follow the steps below.
 
+## DEMO
+Before you start, you can get a feel of the Safe App example. Feel free to skip this step!
+
+The example shows:
+- ✅ How to make a new Safe App.
+- ✅ How to make Swaps, Limit orders and TWAPs.
+- ✅ How to control programatically the swap widget.
+- ✅ How to detect trade parameters (token selection, amounts, etc.).
+- ✅ How to define a dynamic fee.
+- ✅ How to customize the colors, tokens, sounds, etc.
+
+
+To use the demo, just follow the step [Add your custom dapp in Safe Wallet](#add-custom-dapp) bellow, but instead of adding the URL http://localhost:9999, use the URL https://safe.widget.cow.fi.
```

Comment:
```suggestion
To use the demo, just follow the instructions in [Add your custom dapp in Safe Wallet](#add-custom-dapp) below, but instead of adding the URL `http://localhost:9999`, use the URL `https://safe.widget.cow.fi`.
```